examples/webgl_shadowmap_progressive.html

```diff
@@ -101,7 +101,7 @@
 				scene.add( groundMesh );
 
 				// model
-				function loadModel() {
+				async function loadModel() {
```

Comment:
The changes to `webgl_shadowmap_progressive` seem unrelated to the PR.

Reply:
webgl_shadowmap_progressive takes about 5 minutes on its own in the CI system which was why job 7 took 7 minutes and the others about 2-3 minutes. I was testing this example because of this.